db/models.py

```diff
@@ -0,0 +1,23 @@
+from sqlalchemy import Column, Integer, String, Date, ForeignKey
+from sqlalchemy.orm import relationship
+
+from db.engine import Base
+
+
+class DBAuthor(Base):
+    __tablename__ = "author"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String, unique=True)
+    bio = Column(String)
+    books = relationship("DBBook", back_populates="author")
+
+
+class DBBook(Base):
+    __tablename__ = "book"
+
+    id = Column(Integer, primary_key=True, index=True)
+    title = Column(String, unique=True)
+    summary = Column(String)
+    publication_date = Column(Date)
+    author_id = Column(Integer, ForeignKey("author.id"))
```

Comment:
The `DBBook` class should include a relationship back to `DBAuthor` to complete the bidirectional relationship. Consider adding `author = relationship('DBAuthor', back_populates='books')`.